### ISSUE TITLE
Unable to open indexedDB database in your browser. Please make sure you have not disabled access to in-browser storage. 

### ISSUE THREAD

**Issue by mcepl** [KaiRo-at/lantea#3]
That's what I see on https://lantea.kairo.at/ with Firefox 59 on RHEL-7 (that's Linux) ... I don't even know how could I disable IndexedDB. 

Any way how can I find out what's going on, please?

Console shows:
```
Loading failed for the <script> with source “https://lantea.kairo.at//js/piwik.js”.
    lantea.kairo.at:16
starting DB init... ui.js:78:3
error opening mainDB: UnknownError: The operation failed for reasons unrelated
    to the database itself and not covered by any other error code. ui.js:242:5
```

**Comment by KaiRo-at** [comment]
This usually happens because of https://bugzilla.mozilla.org/show_bug.cgi?id=1425146

**Comment by mcepl** [comment]
That's a neat way how to blame an user! I have removed all data related to https://lantea.kairo.at/ and situation remained the same.

**Comment by KaiRo-at** [comment]
Sorry, no intent to blame the user. Unfortunately, I ran into the same issue on almost all devices I have, esp. everywhere I use Nightly (and I also had it on other websites), and I heavily blame the browser, not the user. I found that cleaning all caches in the browser (not cleaning "data") was the only way I could fix the problem, though I spent a few hours trying to fix it from the app side I cfouldn't find any way to do it. In theory I guess I could rename the DB and have all data for all users go away but I'd need to do that repeatedly probably and the real problem is unfortunately in the browser. I am very frustrated about this myself, sorry.
